From bcd84e80be5c3f50462a9161c1aec4ef16a86012 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Sat, 25 May 2024 10:55:26 -0700
Subject: [PATCH] ==== selftests/damon: test DAMOS tried_regions and
 {min,max}_nr_regions ====

Subject: selftests/damon: test DAMOS tried regions

This patch series fixup trivial issues in DAMON selftest code and
implement new kselftests for basic functionality of DAMOS tried regions
and {min,max}_nr_regions feature.  The test for max_nr_regions also test
the recovery from online tuning-caused limit violation.

The first patch fixes a trivial problem in the articial memory access
pattern generator.  Following 3 patches (2-4) implement schemes tried
regions test.  Then a couple of patches (5-6) implementing static
{min,max}_nr_regions functionality test follows.  Final two patches
(7-8) implement dynamic max_nr_regions update test.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/hiIKbBkZ | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/hiIKbBkZ

diff --git a/damon_meta_changes/hiIKbBkZ b/damon_meta_changes/hiIKbBkZ
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.2

